docs/notifications.md

```diff
@@ -85,7 +85,8 @@ Recognized stream names:
  * `notification_from_execution`
    Filter: `contract` field containing a string with hex-encoded Uint160 (LE
    representation) and/or `name` field containing a string with execution 
-   notification name.
+   notification name which should not be longer than runtime.MaxEventNameLen
```

Comment:
The statement is NeoGo-specific, but we want this document to be generic, so we better have some more generic reference. Specific value can be added as well, these constants rarely change.